tools/scripts/authz/dao_genesis_authorizations.json

```diff
@@ -349,5 +349,14 @@
       "msg": "/cosmos.authz.v1beta1.MsgPruneExpiredGrants"
     },
     "expiration": "2500-01-01T00:00:00Z"
+  },
+  {
+    "granter": "pokt10d07y265gmmuvt4z0w9aw880jnsr700j8yv32t",
```

Comment:
Can you please start preparing the next migration (leaving it as a WIP) to make sure we have the authorization there?